Resources/views/ContentType/update_content_type.html.twig

```diff
@@ -25,72 +27,101 @@
 
 {% block content %}
     <section class="ez-serverside-content">
-        Last modified: {{ contentTypeDraft.modificationDate|localizeddate('medium', 'medium', app.request.locale) }}
 
-        {{ form_start(form, {"action": action_url}) }}
+        <div class="context-information">
```

Comment:
we should always prefix the classes with the `ez-` prefix. That said, `context-information` is very generic and not used anywhere in the CSS so it can be omitted (the same goes for `modified` below).